connectors/airwatch/pom.xml

```diff
@@ -160,6 +161,87 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>push-docker-image</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>airwatch-connector</name>
+                                    <alias>airwatch-connector</alias>
+                                    <build>
+                                        <assembly>
+                                            <inline>
+                                                <files>
+                                                    <file>
+                                                        <source>${project.basedir}/src/main/resources/application.yml</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+                                                    <file>
+                                                        <source>${project.basedir}/src/test/resources/docker/managed-apps.yml</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+                                                    <file>
+                                                        <source>${project.basedir}/src/test/resources/docker/application.properties</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+
+                                                </files>
+                                            </inline>
+                                            <targetDir>/opt/vmware/airwatch-connector</targetDir>
+                                        </assembly>
+                                        <from>openjdk:8u151-jre</from>
+                                        <workdir>/opt/vmware/airwatch-connector</workdir>
```

Comment:
It would be good to add a `<healthCheck>` element:
```
<healthCheck>
    <interval>20s</interval>
    <timeout>5s</timeout>
    <retries>6</retries>
    <cmd>curl -f http://localhost:8080/health || exit 1</cmd>
</healthCheck>
```

This won't actually do anything except display health in `docker ps` and `docker inspect`, but it gives users options to do things with that information.

Reply:
Added ```healthCheck``` element.